### DISCUSSION TITLE
pfSense 2.5.0 update broke the syslog pipeline

### DISCUSSION THREAD

**Discussion by SackOfHacks** [Security-Onion-Solutions/securityonion#3057] [Unsupported Versions] [answered]
I noticed that after the pfSense update to 2.5.0, the parsing in the syslog ingest pipeline stopped working.

**Comment by TOoSmOotH** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/pull/3048

Should be addressed in the next release but you can change it now if you want to get it parsed right away.

**Reply by SackOfHacks** [reply]
Thanks for the update. I know the change the parsing needs, what is the correct file to make that change in though. Currently I have a custom syslog file in /opt/so/saltstack/local/salt/elasticsearch/files/ingest/ I suspect there is a better way

**Reply by TOoSmOotH** [reply]
https://github.com/Security-Onion-Solutions/securityonion/pull/3048/files
